Lib/gftools/builder/__init__.py

```diff
@@ -118,6 +122,29 @@ def __init__(self, configfile=None, config=None):
         self.logger = logging.getLogger("GFBuilder")
         self.fill_config_defaults()
 
+    def load_config(self, configfile):
+        with open(configfile) as f:
+            unprocessed_yaml = f.read()
+        try:
+            return load(unprocessed_yaml, schema).data
+        except YAMLValidationError as e:
+            if "unexpected key not in schema" in e.problem:
+                bad_key = str(re.findall(r"'(.*?)'", e.problem))
+                config_keys = re.findall(r'"(.*?)"', str(schema._validator))
+                config_keys.extend(re.findall(r"'(.*?)'", str(schema._validator)))
```

Comment:
Our next task is to remove the regular expressions.

Reply:
This is done in commit 298895b